apps/desktop/src/renderer/ui/patterns/Modal.module.css

```
@@ -0,0 +1,122 @@
+  background: var(--bg-active);
+}
+
+/* ── Body ────────────────────────────────────────────────────────────────── */
+
+.body {
+  padding: var(--space-5);
+}
+
+/* ── Animations ──────────────────────────────────────────────────────────── */
+
+@keyframes fadeIn {
```

Stylelint (17.7.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)